### PR TITLE
Add quick start guide to README to not redirect everyone to the docs

### PR DESCRIPTION
**Pull request by duizendnegen**

### CODE REVIEW

**Comment by LevelbossMike** [comment]
We should maybe also include something about deploy.js etc. so that people get a quick overview how things work together when scanning the readme

**Comment by duizendnegen** [comment]
could do - it does already point to http://ember-cli.github.io/ember-cli-deploy/docs/v0.4.x/configuration/
also the most informative information about deploy.js comes from plugin authors: what kind of parameters they expect

the quick start guide is imo just that: rough guidelines of essentials plus references where to look for further info
if you were to add something about deploy.js, what would you add in? (I find it hard to put together something concise, to tell people about the file, the environments & settings in just a few lines)

**Comment by LevelbossMike** [comment]
yeah true. Just wanted to make sure people are not scared away by the small README if they are too lazy to click on the docs link ;)

File: README.md [hunk 1]
```diff
@@ -9,6 +9,10 @@ Simple, flexible deployment for your Ember CLI app
 npm install ember-cli-deploy --save-dev
 ```
 
+## Quick start
+
+After installation, choose [deployment adapters](http://ember-cli.github.io/ember-cli-deploy/docs/v0.4.x/existing-custom-adapters/) matching your deployment environment, [configure](http://ember-cli.github.io/ember-cli-deploy/docs/v0.4.x/configuration/) your deployment script appropriately and you're ready to [start deploying](http://ember-cli.github.io/ember-cli-deploy/docs/v0.4.x/how-to-use/).
+
 ## Documentation
```
Comment (lukemelia) [inline README.md]: Can we change this heading to "In-depth documentation"? That will help clarify that the nature of "Quick Start".

**Comment by duizendnegen** [comment]
Good idea, it's done

**Pull request merged**